jinseok/model/likeDao.js

```diff
@@ -0,0 +1,74 @@
+const { DataSource } = require('typeorm');
+
+const dataSource = new DataSource({
+  type: process.env.DB_CONNECTION,
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+});
+
+dataSource
+  .initialize()
+  .then(() => {
+    console.log('Data Source has been initialized!');
+  })
+  .catch((err) => {
+    console.error('Error occurred during Data Source initialization', err);
+    dataSource.destroy();
+  });
+
+const createLike = async (userId, postId) => {
+  try {
+    return await dataSource.query(
```

Comment:
```suggestion
    return dataSource.query(
```
어차피 return 문에 작성되는 가장 마지막 코드이기 때문에 비동기처리가 필요하지 않습니다.